app/Console/Commands/RegisterFamiliesFromFile.php

```diff
@@ -105,6 +105,7 @@ public function handle()
                     $registration = new Registration([
                         'fm_chart_on' => ($row->pie_chart == "Y") ? $signup_date : null,
                         'fm_diary_on' => ($row->pie_chart == "Y") ? $signup_date : null,
+                        'fm_privacy_on' => ($row->pie_chart == "Y") ? $signup_date : null,
```

Comment:
Urgh. this makes me sad - is  a bug in the data loader stuff.
The line above yours should be `$row->diary == "Y"` for the diary. 
I've checked the preload data and fortunately it didn't; cause issues as it seems everyone who submitted a pie chart also submitted a diary. However it may be a problem in the future. 

can you
1. amend the diary line in this PR
2. i suggest a `$row->privacy' == "Y"` for the privacy date.

Ta.